Utils/build-release-linux.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -eux
+
+cd "$(dirname $0)"
+
+docker build -f Dockerfile-linux-amd64 . -t itgmania-linux-build:amd64
```

Comment:
Do we need to use docker to build linux releases? Sounds kind of not great to have this as a requirement.

Reply:
It's essential to build the binary on the oldest version of glibc that we want to support, otherwise the binary won't run. *Theoretically* that could be done without docker, but using docker for it is much easier and less error prone.

Reply:
With my comment below, I understand better what's going on. LGTM.